app/lib/importer/crime_incident_reports.rb

```diff
@@ -27,16 +33,22 @@ def import
 
   private
   def import_slice(slice)
+    is_nibrs = @parser.attribution.category.include?("nibrs")
```

Comment:
This awkward conditional is one of those signs that makes me think "this should probably be a subclass instead." But, really, I don't think it's worth putting more effort in right now, given this works fine (and we have test coverage if we want to refactor in the future!)